plugins/svg/src/SvgFeatureRenderer/components/Box.tsx

```diff
@@ -25,7 +25,8 @@ function Box(props: {
   const { feature, region, config, featureLayout, bpPerPx, topLevel } = props
   const { start, end } = region
   const screenWidth = (end - start) / bpPerPx
-  const { left = 0, width = 0 } = featureLayout.absolute
+  const width = (feature.get('end') - feature.get('start')) / bpPerPx
```

Comment:
this is an interesting issue because I was not able to trust the feature layout's calculation of width. it might be that it needs further investigation